tests/test_aggregation.py

```diff
@@ -108,85 +106,3 @@ def test_schema_to_template_ds_attrs():
 
     assert dst.attrs["pangeo-forge:test_attr_key"] == "test_attr_value"
     assert dst.attrs["original_attrs_key"] == "original_attrs_value"
-
-
-def test_concat_accumulator():
```

Comment:
Your changes above make sense but I see you are removing tests here. Should we be keeping some of these?

Reply:
Yeah, that's doable directly via the combinefn. Should be fairly easy...